source/extensionPoints/__init__.py

```diff
@@ -13,6 +13,7 @@
 from logHandler import log
 from .util import HandlerRegistrar, callWithSupportedKwargs, BoundMethodWeakref
 from typing import Set
+from typing import TypeVar, Generic

```

Comment:
these should be joined

```suggestion
from typing import (
	Generic,
	Set,
	TypeVar,
)
```